tchannel-core/src/main/java/com/uber/tchannel/api/Request.java

```diff
@@ -76,18 +82,23 @@ public String toString() {
 
     public static class Builder<U> {
 
+        private static final long DEFAULT_TTL = 100;
```

Comment:
agreed; but I wanted to document the value somehow…I'll just set the TTL to a val and add a wordy-docstring.